pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -69,11 +69,13 @@ const (
 	fileNameEnvoyConfig = "envoy.yaml"
 	fileNameCABundle    = "ca.crt"
 
+	volumeMountPathDevNetTun   = "/dev/net/tun"
 	volumeMountPathCerts       = "/srv/secrets/vpn-server"
 	volumeMountPathTLSAuth     = "/srv/secrets/tlsauth"
 	volumeMountPathDH          = "/srv/secrets/dh"
 	volumeMountPathEnvoyConfig = "/etc/envoy"
 
+	volumeDevNetTun       = "dev-net-tun"
```

Comment:
```suggestion
	volumeNameDevNetTun       = "dev-net-tun"
```

to make the naming identical with the other volume names.

Reply:
Adapted accordingly in latest commit.